internal/venice-common/src/main/java/com/linkedin/venice/meta/DataReplicationPolicy.java

```diff
@@ -30,6 +30,7 @@ public enum DataReplicationPolicy {
    * Samza job per colo pushes to local real-time topic. Leader SNs replicate data from real-time topic in all colos to
    * local version topic.
    */
+  @Deprecated
   ACTIVE_ACTIVE(3);
```

Comment:
Removing this should be fine for ZK. The problem might be in the historical update-store commands in the admin topics.

Reply:
I think removing this might result in errors during dser of znodes?

Reply:
Let's leave it in. If we remove it, we might end up reusing the enum number too

Reply:
Yeah agree